### ISSUE TITLE
Distrcutive componentWillUnmount

### ISSUE THREAD

**Issue by rgoldiez** [VansonLeung/react-native-keyboard-aware-view#1]
I really tried to use this component but spent hours chasing down weird issues when my other keyboard events stopped firing.  I found the culprit.  When this component unmounts, it has removes **ALL** keyboard listeners.  This is problematic if you are using Navigator, side menu/drawers, etc.  This component should really only remove the listeners it added, not all listeners.

```
  componentWillUnmount() {
    DeviceEventEmitter.removeAllListeners('keyboardWillShow');
    DeviceEventEmitter.removeAllListeners('keyboardWillHide');
  }
```

**Comment by VansonLeung** [comment]
@rgoldiez oh... thanks, I will fix it now

**Comment by VansonLeung** [comment]
Just Fixed in version 0.0.5, thanks please try again.
